docs/Agent/user-guide/man_page.md

```diff
@@ -377,7 +371,7 @@ Show this message and exit.
 ---
 
 **NAME**  
-`pbench-results-move`, Move results tarball to the server to a 1.0 Pbench Server.
+`pbench-results-move` - move results tarball to the server to a 1.0 Pbench Server
```

Comment:
I think it would be best if we didn't mention tar, tar archives, or "tarballs".  The command description mentions "result directories"...it would be better to use that terminology here.

Reply:
Thanks for being so thorough executing this change!